### PR TITLE
Dedicated InfluxQL Python instructions

### PR DESCRIPTION
**Pull request by sanderson**
- Adds instructions for using the `influxdb3-python` client library to query InfluxDB with InfluxQL
- Updates the existing Python SQL query guide to use the `influxdb3-python` client

---

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+The `influxdb_client_3` module provides a simple and convenient way to interact
+with {{< cloud-name >}} using Python. This module supports both writing data to
+InfluxDB and querying data using SQL or InfluxQL queries.
+
+{{% note %}}
+_To query data with **InfluxQL** and Python, see
+[Use InfluxQL with Python](/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python/)._
+{{% /note %}}
+
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python.md:204]:
```suggestion
```

Self-referencing? Is it supposed to be SQL?

File: content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+#### Example {#execute-query-example}
+
+{{% code-placeholders "DATABASE_(NAME|TOKEN)" %}}
+```py
+# query-example.py
+
+from influxdb_client_3 import InfluxDBClient3
+
+client = InfluxDBClient3(
+    host='b0c7cce5-8dbc-428e-98c6-7f996fb96467.a.influxdb.io',
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python.md:271]:
```suggestion
    host='cluster-id.influxdb.io',
```

File: content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+In your editor, copy and paste the following sample code to a new file--for
+example, `query-example.py`:
+
+{{% code-placeholders "DATABASE_(NAME|TOKEN)" %}}
+```py
+# query-example.py
+
+from influxdb_client_3 import InfluxDBClient3
+
+# Instantiate a InfluxDBClient3 configured for your database
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python.md:232]:
```suggestion
# Instantiate an InfluxDBClient3 configured for your database
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python.md]: Ah thanks. Copy-pasta.

**Pull request merged**